CMakeLists.txt

```diff
@@ -119,6 +119,17 @@ else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -Wextra -Wno-unknown-pragmas -std=c++11")
 endif()
 
+add_definitions(
```

Comment:
This block is already set in `src/cpp/CMakeLists.txt`. Is there some other piece failing besides the main library?

Reply:
Done in https://github.com/eProsima/Fast-RTPS/pull/229/commits/19f5566173aff200c0e8733f74cc1583fa174490

Reply:
In https://github.com/eProsima/Fast-RTPS/blob/master/src/cpp/CMakeLists.txt#L213 you should add the c++14 check.

Reply:
I had already removed that in 19f5566

Reply:
@esteve I'm rebasing certain code over your PR and I see this block was introduced again. I didn't check it again after your rebase. Could it be an error during your rebase? Can I remove this block?

Reply:
That's strange. Yeah, just remove it, it should work without it.